language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/sourceprune/RHSCompletionsTokenTraverser.java

```diff
@@ -29,7 +32,7 @@
  * 
  * @since 0.995.0
  */
-class RHSTokenTraverser extends AbstractTokenTraverser {
+public class RHSCompletionsTokenTraverser extends AbstractTokenTraverser {
```

Comment:
Is it possible to make the class package-private since we do not expose this as a public API?